benchmarks/benchmark_dataset.py

```diff
@@ -771,6 +771,58 @@ def sample(self,
         return sampled_requests
 
 
+# -----------------------------------------------------------------------------
+# MT-Bench Dataset Implementation
+# -----------------------------------------------------------------------------
+
+
+class MTBenchDataset(HuggingFaceDataset):
+    """
+    MT-Bench Dataset.
+    https://huggingface.co/datasets/philschmid/mt-bench
+
+    We create a single turn dataset for MT-Bench. 
+    This is similar to Spec decoding benchmark setup in vLLM
+    https://github.com/vllm-project/vllm/blob/9d98ab5ec/examples/offline_inference/eagle.py#L14-L18 # noqa: E501
+    """
+
+    DEFAULT_OUTPUT_LEN = 256  # avg len used in SD bench in vLLM
```

Comment:
QQ: Which SD bench do you mean?

Reply:
I was refering to the [offline eagle bench](https://github.com/vllm-project/vllm/blob/9d98ab5ec60ae43453e3bd2e68452483b3ced199/examples/offline_inference/eagle.py#L93). Lmk if you would like me to clarify this in the comment

Reply:
oh I think it's quite random then. What about using longer outputs like 1K+?

Reply:
1K would make the MTBench run 4x longer. My experience so far has been that 256 is good enough to know the inference metrics and doesnt change much with longer output len with MTBench.